src/cmd/ksh93/include/defs.h

```diff
@@ -573,4 +572,10 @@ static struct lconv *lp;
 
 #define _pipe_socketpair    1   /* use socketpair() for peekable pipe() */
 
+/*
+ * SHELLMAGIC is enabled if operating system supports shebang
+ * TODO: Shall we remove this macro from sources ?
```

Comment:
Yes, this should be removed. This hasn't been an issue for nearly four decades. See https://en.wikipedia.org/wiki/Shebang_(Unix). Even when I got my first hands on experience with UNIX around 1985 kernel shebang support was widespread.